### ISSUE TITLE
Bugs: Error on uploading any svg files

### ISSUE THREAD

**Issue by rizesky** [pqina/filepond-plugin-image-transform#14]
<img width="960" alt="error_upload_svg" src="https://user-images.githubusercontent.com/34252005/72965198-43f71b00-3dee-11ea-8612-e562834bb027.PNG">

Hi im trying to upload svg, with following registration:
<img width="357" alt="config" src="https://user-images.githubusercontent.com/34252005/72965327-9afcf000-3dee-11ea-970b-fef76cf8af5b.PNG">

I always got error,
What should i do to solve this?
thank you

**Comment by rikschennink** [comment]
I'm not sure, please create a public test case on codesandbox.io

**Comment by kylefox** [comment]
We're running into this error as well. All SVGs break our FilePond instances. I created a minimal example to demo the problem.

* Test case: https://kylefox.ca/filepond-demo/
* Code: https://github.com/kylefox/filepond-demo

To reproduce, drag any SVG file into the filepond. After a moment or two, you'll see this error:

```
main.js:13 Uncaught TypeError: Cannot read property 'aspectRatio' of undefined
    at FileReader.C.u.onloadend (main.js:13)
```

**Comment by rikschennink** [comment]
@kylefox thanks Kyle! I've confirmed the issue, fixing now.

**Comment by rikschennink** [comment]
Fixed in version `3.7.2`

**Comment by kylefox** [comment]
Awesome, I can confirm the fix worked 👍 Thanks for the quick response! You can likely close this issue now.
